docker/base/Dockerfile

```diff
@@ -49,26 +48,28 @@ RUN yum -y update \
     man-db \
     lz4-devel \
     jq \
-    # Flux-sched dependencies
-    libboost-graph-devel \
-    libboost-system-devel \
-    libboost-filesystem-devel \
-    libboost-regex-devel \
+    # Flux-sched Dependencies
+    boost-devel \
+    boost-graph \
+    boost-system \
+    boost-filesystem \
+    boost-regex \
```

Comment:
This is not your fault but `flux-sched` recently replace `readline-devel` with `libedit-devel`, could you replace this depedency?

Reply:
tried to have base dockerfile without `readline-devel` and got the following error (while creating dockerfile for `flux`):
```
#6 10.54 checking for readline in -lreadline... no
#6 10.58 configure: error: Please install GNU readline
------
executor failed running [/bin/sh -c V=$FLUX_SCHED_VERSION    && PKG=flux-sched-$V    && 
URL=https://github.com/flux-framework/flux-sched/releases/download    && 
wget ${URL}/v${V}/${PKG}.tar.gz    && tar xvf ${PKG}.tar.gz    && cd ${PKG}    && 
PYTHON_VERSION=3 ./configure --prefix=/usr    && make -j 4    && make install    && cd ..    && 
rm -rf flux-sched-*    && ldconfig]: exit code: 1
```

Reply:
This is grabbing an older version of flux-core/sched.  I bump the versions in #51 and make the associated dependency change.  So I think we can skip that change in the PR for now.